docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -262,6 +267,53 @@ docker run -i --rm -p 8080:8080 quarkus-quickstart/getting-started
 
 NOTE: Interested by tiny Docker images, check the {quarkus-images-url}/graalvm-{graalvm-version}/distroless[distroless] version.
 
+== Producing a container with a multi-stage Docker build
+
+On the previous paragraph, we show you how to build a native image with Maven, however this implies that GraalVM is installed locally, and if you use
+a CI/CD pipeline, inside your CI/CD.
+
+If not, you can use Docker to make the Maven build by using a multi-stage Docker build. A multi-stage Docker build is like two Dockerfile files combined in one, the first is used to build the artifact used by the second. 
+
+In this guide we will use the first stage to generate the native image using Maven and the second stage to create our runtime image.
+
+[source,dockerfile]
+----
+## Stage 1 : build with maven builder image with native capabilities
+FROM quay.io/quarkus/centos-quarkus-maven:graalvm-{graalvm-version} AS build
+COPY src /usr/src/app/src
+COPY pom.xml /usr/src/app
+RUN mvn -f /usr/src/app/pom.xml -Pnative clean package
+
+## Stage 2 : create the docker final image
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+WORKDIR /work/
+COPY --from=build /usr/src/app/target/*-runner /work/application
+RUN chmod 775 /work
+EXPOSE 8080
+CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]
+----
+
+
+Save this file in `src/main/docker/Dockerfile.multistage` as it is not included in the getting started quickstart.
+
+[WARNING]
+====
+Before we can launch our Docker build, we need to update the default `.dockerignore` file as it filters everything except the `target` directory and as we plan to build inside a container we need to be able to copy the `src` directory. So edit you `.dockerignore` and remove or comment all its lines.
```

Comment:
```suggestion
Before we can launch our Docker build, we need to update the default `.dockerignore` file as it filters everything except the `target` directory and as we plan to build inside a container we need to be able to copy the `src` directory. So edit your `.dockerignore` and remove or comment its content.
```

Reply:
Also `Before we can launch` -> Before launching`

Reply:
done